katas/content/distinguishing_unitaries/h_x/solution.md

```diff
@@ -0,0 +1,8 @@
+> As a reminder, $$H = \frac{1}{\sqrt2} \begin{bmatrix} 1 & 1 \\ 1 & -1 \end{bmatrix}$$
+
+The solution relies on the well-known identity $HXH = Z$ (i.e., Hadamard gate can be used to convert $X$ to $Z$ and vice versa). Applying a sequence of gates "given unitary - $X$ - unitary" will be equivalent to $XXX = X$ gate if the unitary was $X$, and to $HXH = Z$ if the unitary was $H$. With this observation we need to distinguish $X$ from $Z$, which can be done using the $|0\rangle$ state: $X$ gate will change it to $|1\rangle$, and $Z$ gate will leave it unchanged.
```

Comment:
We did a massive sweep of the katas about two weeks ago to use $\ket{...}$ instead of $|...\rangle$ (found a few actual bugs in the process, so it's a good thing to have done). Let's stick to the new formatting convention here, and use $\ket{...}$ here and in the rest of the tasks (though z_s doesn't use kets I think)

Reply:
good call, will replace those